CRM/Contribute/DAO/ContributionRecur.php

```diff
@@ -369,7 +372,7 @@ public static function &fields() {
         'start_date' => [
           'name' => 'start_date',
           'type' => CRM_Utils_Type::T_DATE + CRM_Utils_Type::T_TIME,
-          'title' => ts('Recurring Contribution Started Date'),
+          'title' => ts('Start Date'),
```

Comment:
The reason for these ridiculously long names may have been to disambiguate fields on the import/export forms where they are all shown in a flat list.